#!/usr/bin/env python
PACKAGE = "project_pointcloud"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()
gen.add("x_adjust", double_t, 0, "A double parameter", 0.0, -1.0, 1.0)
gen.add("y_adjust", double_t, 0, "A double parameter", 0.0, -1.0, 1.0)
gen.add("z_adjust", double_t, 0, "A double parameter", 0.0, -1.0, 1.0)
gen.add("roll_adjust", double_t, 0, "A double parameter", 0.0, -1.0, 1.0)
gen.add("pitch_adjust", double_t, 0, "A double parameter", 0.0, -1.0, 1.0)
gen.add("yaw_adjust", double_t, 0, "A double parameter", 0.0, -1.0, 1.0)
exit(gen.generate(PACKAGE, "project_pointcloud", "adjust_"))